app/src/main/java/ch/epfl/sdp/blindwar/game/util/Util.kt

```diff
@@ -69,10 +67,25 @@ object Util {
         }
     }
 
-    fun updateProfileImage(liveData: LiveData<StorageReference>,
-                           imageView: ImageView,
-                           viewLifecycleOwner: LifecycleOwner,
-                           context: Context) {
+    /**
+     * Check if internet is working via one ping
+     *
+     * @return
+     */
+    fun isOnline(): Boolean {

```

Comment:
Useful function ! We should make use of it to create a NetworkConnectivtyChecker that would signal the current connectivity state of an user.